src/org/labkey/test/tests/assay/FolderImportAssayRenameTest.java

```diff
@@ -0,0 +1,152 @@
+package org.labkey.test.tests.assay;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.Assays;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.pages.ReactAssayDesignerPage;
+import org.labkey.test.util.core.webdav.WebDavUploadHelper;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+@Category({Assays.class, Daily.class})
+public class FolderImportAssayRenameTest extends BaseWebDriverTest
+{
+
+    private static final String ORIGINAL_PROJECT = "Folder_Import_Assay_Rename_Test";
+    private static final String SECOND_PROJECT = "Folder_Import_Assay_Rename_Test_Copy";
+
+    private static final File GPAT_ASSAY_XLS = TestFileUtils.getSampleData("GPAT/trial01.xls");
+
+    @BeforeClass
+    public static void doSetup()
+    {
+        FolderImportAssayRenameTest init = (FolderImportAssayRenameTest) getCurrentTest();
+
+        init._containerHelper.createProject(ORIGINAL_PROJECT, "Assay");
+        init._containerHelper.createProject(SECOND_PROJECT, "Assay");
+
+        init.goToProjectHome();
+    }
+
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("assay");
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return ORIGINAL_PROJECT;
+    }
+
+    @Override
+    public void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        _containerHelper.deleteProject(ORIGINAL_PROJECT, afterTest);
+        _containerHelper.deleteProject(SECOND_PROJECT, afterTest);
+    }
+
+    /**
+     * <p>
```

Comment:
I appreciate the effort to add this test! I was thinking this test scenario can piggyback on one of the existing AssayExportImportTest: modify one of the test to do a simple assay creation after import. Though assay design rename is a new feature, I worry after a while this test (especially with the file name mentioning rename, yet the test doesn't do renaming) will feel odd. It's good to have some coverage, but I think having a dedicated test class for it is probably not necessary and a bit confusing.